playbooks/roles/ocim/meta/main.yml

```diff
@@ -1,3 +1,20 @@
 dependencies:
+
   - name: pmbauer.tarsnap
     when: "{{ OPENCRAFT_BACKUP_SWIFT_ENABLED }}"
+
+  - name: nginx-proxy
```

Comment:
Is `allow_duplicates` true by default? If not I believe we'll need to add it into `nginx-proxy`'s meta.

Reply:
As always, Ansible semantics are weird.  Roles with different parameters won't be deduplicated, regardless of `allow_duplicates`, and running nginx-proxy twice with the same parameters is indeed pointless, so we don't need that flag.  Furthermore, it works, and also worked for the prometheus role, which includes nginx-proxy twice as well.